### DISCUSSION TITLE
Viðbætur í glósur um PostgreSQL

### DISCUSSION THREAD

**Discussion by snaeja13** [HI-IDN/IDN302G-GitBook#3] [Hugmyndir]
Gæti verið sniðugt að bæta við í glósurnar undir **PostgreSQL tenging** nánari upplýsingar hvernig eigi að fá IDE og tengjast PostgreSQL. Í rauninni bara fara í gegnum skrefin sem við fórum í gegnum í tíma upp á töflu. Væri hægt að setja inn skjáskot af skrefunum. Með þessari viðbót gætu þá vonandi allir sett þetta upp sjálfir í tölvunni sinni út frá glósunum.

**Comment by tungufoss** [comment]
Þá svona? https://tungufoss.gitbook.io/hi-idn302g/sql-grunnatridi/database_interaction#ide-fyrir-gagnagrunnskerfi 
<img width="573" alt="{969A5326-481C-4C02-8615-29CAE120970F}" src="https://github.com/user-attachments/assets/99af29f6-0bdf-43ab-96cf-10972bae336c">

**Comment by tungufoss** [comment]
Takk fyrir ábendinguna! Tekin áfram í issue #53 svo hægt sé að vinna úr henni. Loka þessari umræðu sem afgreiddri.
